conda/plugins/manager.py

```diff
@@ -159,6 +161,24 @@ def get_solver_backend(self, name: str = None) -> type[Solver]:
 
         return backend
 
+    def get_pre_command_hook_actions(
```

Comment:
Technically a generator shouldn't use “get” but this:
```suggestion
    def yield_pre_command_hook_actions(
```